Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2457,6 +2460,48 @@ void LocalDOMWindow::finishedLoading()
     }
 }
 
+LocalDOMWindow::PerformanceEventTimingCandidate LocalDOMWindow::initializeEventTimingEntry(const Event& event, EventTypeInfo typeInfo)
+{
+    LOG_WITH_STREAM(PerformanceTimeline, stream << "Initializing event timing entry of type " << event.type());
+    // FIXME: implement InteractionId logic
+    // event.timeStamp() may not be reliable (wall clock):
+    auto startTime = std::min(event.timeStamp(), MonotonicTime::now());
+    return PerformanceEventTimingCandidate {
+        .typeInfo = typeInfo,
+        .cancelable = event.cancelable(),
+        .startTime =  performance().relativeTimeFromTimeOriginInReducedResolution(startTime),
+        .processingStart = performance().now(),
```

Comment:
This will invoke `MonotonicTime::now` again inside for no good reason. We should be calling `MonotonicTime::now()` exactly once.